toxicity/preprocessing.py

```diff
@@ -191,9 +191,54 @@ def tf_idf(train, test, params=None, remove_numbers_function=True, debug=False):
     if remove_numbers_function:
         train, test = remove_numbers(train, test)
 
-    re_tok = re.compile(f'([{string.punctuation}“”¨«»®´·º½¾¿¡§£₤‘’])')
-
-    def tokenizer(s): return re_tok.sub(r' \1 ', s).split()
+    if stemming:
```

Comment:
From the code it seems that at most one of `stemming` and `lemmatization` can be `True` (which I think is the correct approach). Can we make sure the user did not mistakenly pass both of them as True? For example something like:

```
if lemmatization + stemming == 2:
  raise ValueError("At most one...")
```